app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2019 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <color name="colorBackground">#161f1a</color>
```

Comment:
Our recommendation is instead of varying colors in the night configuration, create an alternative of your theme in `values-night` which sets different colors.

Reply:
Still outstanding.

Reply:
Done! Added benefit of removing duplicate color names that map to the same color.

I wasn't completly sure how to set the color suffixes and the exact formula (should these be _200 or _300? Is the baseline always _500?).  LMK if this is completely off the mark?